Add kinopub

DanielLavrushin opened the pull request:
add `kino.pub` domains set

IceCodeNew commented:
WIP

DanielLavrushin commented:
Hey @IceCodeNew do you want to add/fix something in the PR or actually it can be merged?

IceCodeNew commented on data/kinopub:
```diff
@@ -0,0 +1,8 @@
+m.pushbr.com
+mos-gorsud.co
+zamerka.com
+regexp:(\w+)-static-[0-9]+\.cdntogo\.net$
```
if the cdn domains can be enumerated, it is recommended not to use regexp.
Either wise, pls provide more info so I can confirm whether these domains are belong to kinopub

IceCodeNew commented on data/kinopub:
```diff
@@ -0,0 +1,8 @@
+kino.pub
+kinopub.online
+gfw.ovh
+vjs.zencdn.net
```
try to maintain this list like:

```
kino.pub
kinopub.online

# CDN
vjs.zencdn.net
abc-static-123.cdntogo.net
```

IceCodeNew commented:
more to do:
1. sort
2. provide background info about each domains

DanielLavrushin commented:
@IceCodeNew  done! Its been a while since I added this, so I dont remember already about couple of domains, so I have removed them from the list :).

Added the comments. Changes to domain and resource tracking:

* Added primary domains: `kino.pub` and `kinopub.online`.
* Included mirror sites: `gfw.ovh` (subdomains) and `mos-gorsud.co` (used to generate mirrors via `gfw.ovh`).
* Added a regular expression to identify KinoPub's CDN servers (`(\w+)-static-[0-9]+\.cdntogo\.net
).
